testsuite/tests/typing-misc/pr6416.ml

```diff
@@ -27,13 +27,9 @@ Error: Signature mismatch:
        is not included in
          val f : t/2 -> unit
        Line 6, characters 4-14:
-      type t = B
-      ^^^^^^^^^^
-Definition of type t/1
-Line 2, characters 2-12:
-    type t = A
-    ^^^^^^^^^^
-Definition of type t/2
```

Comment:
Maybe you have said it before, but how come the definitions are not printed anymore? Arguably that is a regression for the message, but maybe that comes from (otherwise sensible) `expect` vs. batch-mode difference.

Reply:
Yes it is buried towards the end of the first message:

>a regression when displaying name conflicts in the terminal (there is no highlighting anymore). I plan to fix that in an upcoming PR which will turn such conflicts into proper sub-messages (I could do that here but it involves a whole bunch of uninteresting plumbing)